packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useBlocksStore.js

```diff
@@ -311,6 +312,10 @@ export function useBlocksStore() {
       },
       matchNode: (node) => node.type === 'code',
       isInBlocksSelector: true,
+      handleEnterKey(editor) {
+        // Insert a new line within the block
+        Transforms.insertText(editor, '\n');
```

Comment:
I'm not sure if `\n` is the best way to represent a new line within a block, since it seems specific to HTML

Reply:
I think there is also `Editor.insertBreak(editor);` as an option but for your purposes I presume your solution is better

Reply:
it is definitily better your solution